polyfill/package.json

```diff
@@ -50,11 +50,14 @@
     "codecov": "^3.6.5",
     "core-js": "^3.6.4",
     "full-icu": "^1.3.0",
+    "lcov-result-merger": "^3.1.0",
```

Comment:
Can we use https://docs.codecov.io/docs/flags instead of merging the lcov files locally?

Reply:
I found you can just submit two lcov files to codecov.io, so we don't need to merge them locally after all.

Flags, if I understand the documentation correctly, would be for if we wanted separate coverage statistics for the demitasse and test262 tests... but maybe that's what you were referring to?

Reply:
I think it would be useful to have the separate coverage as well, yeah.

Reply:
OK, I think I've got this working now.

Flags seem to be full of bugs, though. There's a notice on the site saying they are not displayed in the overview because they're being refactored. The status for each flag in the [autogenerated comment](https://github.com/tc39/proposal-temporal/pull/423#issuecomment-596848074 above seems correct, but the pull request statuses at the bottom seem wrong. Oh well, if it gets too annoying we can always remove them.